e2e/cypress/integration/messaging/markdown_spec.js

```diff
@@ -88,7 +88,7 @@ describe('Markdown', () => {
         });
     });
 
-    it('MM-T1744: Markdown tables', () => {
+    it('MM-T2244 Markdown tables', () => {
```

Comment:
This is a typo error. Correct key is [MM-T2244](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T2244) (see related PR [here](https://github.com/mattermost/mattermost-webapp/pull/7146)).

[MM-T1744](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T1744) is already automated [here](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/integration/interactive_menu/select_with_keys_spec.js#L50).